pkgs/development/libraries/protobuf/generic-v3.nix

```diff
@@ -30,7 +30,7 @@ mkProtobufDerivation = buildProtobuf: stdenv: stdenv.mkDerivation {
       --replace 'tmpnam(b)' '"'$TMPDIR'/foo"'
   '';
 
-  nativeBuildInputs = [ autoreconfHook buildPackages.which buildPackages.stdenv.cc buildProtobuf ];
+  nativeBuildInputs = [ autoreconfHook which buildPackages.stdenv.cc buildProtobuf ];
```

Comment:
What is the reason for not dropping `buildPackages` from `buildPackages.stdenv.cc`?

Reply:
none unless I forgot something how stdenv works.